examples/classic-openshift-multi-zone/variables.tf

```diff
@@ -157,7 +157,7 @@ variable "tags" {
 
 variable "subnet_id" {
```

Comment:
Same as above. Just check, are we using this parameter on module side.

Check where ever we are using the subnet_id, whether its required or not.